index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>Practice Website</title>
+  </head>
+
+  <body>
+    <div class="top">
```

Comment:
Your would look more clean if you replaced this div with the [header] semantic element and then close it off at the "spotlight" div